### DISCUSSION TITLE
Adding multiple cameras on the Wyze Bridge

### DISCUSSION THREAD

**Discussion by praveenvj1979** [mrlt8/docker-wyze-bridge#1033] [Q&A]
I had already set up Wyze bridge to capture the feed from my Wyze CAM v3. Now added a Pan CAM V3 to my house. However when restarting the Docker Container, it still detects only 1 camera. How can I configure the Docker Container to detect both cameras?

**Comment by mrlt8** [comment]
The bridge tries to cache the camera data locally to avoid pinging the wyze servers, but you should be able to clear it in the webUI by selecting the "restart" -> "clear cache and reconnect".
